src/screens/Library.jsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from "react";
+import likesStyles from "./Likes.module.css";
+import profileStyles from "./Profile.module.css";
+import Card from "../components/Card";
+import { API } from "../static/Services";
+import Loader from '../components/Loader';
+import buttonStyles from "../components/Button.module.css";
+
+const defaultState = {
+    labs: [],
+    candidates: [],
+    loading: <Loader />,
```

Comment:
This is a bit uncommon. You usually don't keep components as part of the state. It happens on extremely rare occasions. 

Instead, you would use a short circuit operator, as I've seen you used on some other lines, and you would render the <Loader/> if there is no data. Check comment on line 92 for an example

Reply:
Yes, I don't know why I did this. I'm gonna change it, thanks!